src/components/App.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import UserInfo from "./user/userComponent";
+
+const App = () => <UserInfo />;
```

Comment:
Instead of creating a new `src/component/App.js`, delete it and you can add a new `Route` in the `src/routes.js` file where you can add the `UserInfo` component too.

Reply:
I have succeeded in changing the files thank you